lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -621,6 +637,34 @@ def _split(self, node):
 
         return before, after
 
+    def sample_from_channel(self, channel_slice, include_dead_time=True) -> Slice:
```

Comment:
since this is effectively another downsampling method, I think it would be good to include an optional `reduce` function with `np.mean` as the default like we do for `Slice.downsample_over()`

Reply:
Yeah, I was doubting about that. I'll add it.